a10networks/a10-octavia
# STACK-1158 STACK-1159 STACK-1161 STACK-1163 VE VLAN support for hardw…

TestCases:
1) Add 1 VIP and 2 Members in 3 different VLANs and send traffic
2) Delete the Members and VIP one by one and check the VLANs and VE deleted
3) Add VIP and Member from same VLAN and another member in different VLAN send traffic
4) After above case Delete the Member on VIP VLAN and send traffic
5) With network_type flat add 1 VIP and 2 Members and send traffic.
6) With above setup remove 1 Member and send traffic.
7) With above setup remove the remaining entities and check configuration.

Config Sample:
[a10_global]
network_type = "vlan"

[rack_vthunder]
devices=[
 {
  "project_id" : "2ff0ac52336d4b868ef1042fc69245ef",
  "username" : "admin",
  "password" : "password",
  "ip_address" : "10.0.0.72",
  "device_name" : "rack_vthunder"
 }]
